REST/PowerShell/Feeds/SyncPackages.ps1

```diff
@@ -137,6 +137,16 @@ $destinationSpaceId = ((Invoke-RestMethod -Method Get -Uri "$destinationOctopusU
 
 # Create HTTP clients 
 $httpClientTimeoutInMinutes = 60
+Try { 
+    Add-Type -AssemblyName System.Net.Http
+    $client = [System.Net.Http.HttpClient]::new()
+}
+Catch {
+    Write-Warning "Something went wrong when trying the command:"
+    Write-Warning "Add-Type -AssemblyName System.Net.Http"
+    Write-Warning "`$client = [System.Net.Http.HttpClient]::new()"
+    Write-Host "Continuing..."
```

Comment:
Should this just continue? What are the impacts of not stopping at this point?

Reply:
The next line after Continuing will error with `New-Object : Cannot find type [System.Net.Http.HttpClientHandler]: verify that the assembly containing this type is loaded.`

Maybe this is unfounded, but my thinking was that the only reason why this would fail would be permissions related. You can run the two lines ad nauseam as they generate no output and no issue running the command repeatedly. If `Add-Type -AssemblyName System.Net.Http` were to fail, it is also possible that the `System.Net.Http` may already exist as required to run the rest of the script without issue. I'm not an expert, but my understanding is that this is native in PowerShell Core 6+. I ran into the error in PS5.1 on a Server2022 box.

tl;dr - I'm happy to remove the try/catch if you think it is unnecessary